### DISCUSSION TITLE
Default build_log_retention option?

### DISCUSSION THREAD

**Discussion by giva01121** [concourse/concourse#6337] [Help & Support] [answered]
Hello,
is there a way to make a global default option for max build log retention? I see that in the past it was possible via this PR:
https://github.com/concourse/concourse/pull/2170
But the spec file no longer exists

**Comment by chenbh** [comment, marked as answer]
Answered in https://github.com/concourse/concourse/pull/2170#issuecomment-737205395. This repo used to be only used for bosh related stuff and the actual code lived elsewhere, but now the `concourse/concourse` repo is only for code and the release repos are moved to https://github.com/concourse/concourse-bosh-release and https://github.com/concourse/concourse-chart

**Reply by giva01121** [reply]
> Answered in [#2170 (comment)](https://github.com/concourse/concourse/pull/2170#issuecomment-737205395). This repo used to be only used for bosh related stuff and the actual code lived elsewhere, but now the `concourse/concourse` repo is only for code and the release repos are moved to https://github.com/concourse/concourse-bosh-release and https://github.com/concourse/concourse-chart

@chenbh thanks, for your reply. Once we configure this global setting is it possible to be ignored if we set a local setting in the pipeline? Or either global retention or pipeline specific retention?
